### ISSUE TITLE
Static groups from 3.3 converted to dynamic ones in 3.4dev

### ISSUE THREAD

**Issue by AEgit** [JabRef/jabref#1434]
JabRef version 3.4dev--snapshot--2016-05-18--master--f4270a6 on Windows 10
(same issue is also found in JabRef version 3.4dev--snapshot--2016-05-18--master--2bd4c4a)

It seems that static groups of the current stable branch of JabRef 3.3 are converted to dynamic ones, in the new developer version 3.4

Steps to reproduce:

1. Open a Jabref database from the current stable release (3.3)
2. Previous static groups are displayed in italic (like dynamic groups)
3. Rightclick --> Edit group --> Shows the group as "Dynamically group entries by searching a field for a keyword"
4. The "Keyword" is the group name

**Comment by hudcap** [comment]
I am having the same issue, having loaded my v3.3 db into v3.4 stable.
Furthermore, when I change a group to static, it does not persist.

**Comment by algom** [comment]
I am also having the same problem, which went through to the final JabRef 3.4, all groups from a JabRef 3.3 database are converted to dynamic groups, now group entries are italicized and when an entry is selected is also underlined (italicized and underlined), design-wise a no go from any typographic point of view. I also tried starting fresh with JabRef 3.4 and the same is obtained. I created a dummy database and 4 "static" groups. After creating each group immediately they are converted to dynamic (See screenshot), editing them back to static does not make a change.

Besides this, when group view is toggled along with the Web search, they go to opposite sides of the page, and can no longer be reduced in height, creating a huge space if only few groups are in the database (See screenshot), if the up and down arrows are used, only the Web search box moves to the top, but cant be arranged just below the groups which might be useful to have everything in reach and not in the bottom, as well as the one or more line toggles in the group interface that allowed to have a compact group interface if one has many groups and only needs a few.

![untitled](https://cloud.githubusercontent.com/assets/9680944/15903199/c50b8e9e-2dab-11e6-8e42-7a9d59452872.png)
